pkg/server/bundle/client/manager.go

```diff
@@ -350,8 +351,6 @@ func calculateNextUpdate(b *spiffebundle.Bundle) time.Duration {
 
 func cloneTrustDomainConfigs(configs map[spiffeid.TrustDomain]TrustDomainConfig) map[spiffeid.TrustDomain]TrustDomainConfig {
 	clone := make(map[spiffeid.TrustDomain]TrustDomainConfig, len(configs))
-	for k, v := range configs {
-		clone[k] = v
-	}
+	maps.Copy(clone, configs)
 	return clone
 }
```

Comment:
It think we might be able to replace this whole function with `maps.Clone()`.

Reply:
Doing this actually turns out to cause a test failure, as it changes behaviour: `maps.Clone()` returns `nil` for a `nil` map. So holding off on making those changes.